lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -624,7 +624,9 @@
     <datatype extension="obfs" type="galaxy.datatypes.molecules:OBFS" mimetype="text/html" display_in_upload="true"/>
     <datatype extension="drf" type="galaxy.datatypes.molecules:DRF" display_in_upload="true"/>
     <datatype extension="phar" type="galaxy.datatypes.molecules:PHAR" display_in_upload="false"/>
-    <datatype extension="pdb" type="galaxy.datatypes.molecules:PDB" display_in_upload="true"/>
+    <datatype extension="pdb" type="galaxy.datatypes.molecules:PDB" display_in_upload="true">
```

Comment:
Cool, can we enable this for SDF and MOL as well?

Reply:
Sure. Should be done with https://github.com/galaxyproject/galaxy/pull/8347/commits/0b90afcfcbd14a5bfa67746f1c32cd94ad0dd0a0 (although I didn't actually have test files to try out).